src/js/pages/jobs/JobDetailPage.js

```diff
@@ -237,24 +237,23 @@ class JobDetailPage extends mixin(TabsMixin) {
   }
 
   render() {
-    // TaskDetailView
     if (this.props.params.taskID) {
       return this.props.children;
     }
 
-    const { job, jobTree } = this.props;
+    const { job } = this.props;
 
     return (
       <Page>
         <Page.Header
           actions={this.getActions()}
-          breadcrumbs={<JobsBreadcrumbs tree={jobTree} item={job} />}
+          breadcrumbs={<JobsBreadcrumbs item={job} />}
           tabs={this.getTabs()}
         />
         {this.tabs_getTabView(job)}
         <JobFormModalContainer
           isEdit={true}
-          job={job}
+          job={new Job(JSON.parse(job.json))}
```

Comment:
Not sure why do we need this?

Reply:
Asking for a friend who is currently working on `JobFormModalContainer` :D

Reply:
@nLight  I think we need it right now to not rework the entire `JobFormModalContainer` as this is something I would expect to happen in your work. Am I wrong here to cut the scope?

Reply:
I'm not following. Why is it needed to rework something there?

Reply:
Okay, as far as I understand it we have a JIRA ticket that says "Implement mediator for JobsModalContainer" and I understood that we would go ahead there in the ticket and remove the dependency to the Job struct there. Therefore I decided to not touch it here

Reply:
The old `JobDetailContainer` passed a `Job (Struct)` to the `JobFormModalContainer`. We dont have `Job (Struct)` anymore, so they are trying to regenerate the old one from the new `Job` type. You have to redo your `JobmodelFormContainer` before they are able to pass in the new `Job` type. right now this didnt happen.

Reply:
I guess this blocks this PR, Dima has to finish the reworking before this can get merged 🤔

Reply:
Hm, I wasnt abe to find anthing that is broken because of this branch. 

But I'm not able to create/edit/delete labels - here and on master /cc @nLight